mylandingpage.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="C:\Users\U A E\Documents\desktop files\Desktop\my tasks\my landing page\sytle3.CSS">
+    <title>My landing page</title>
+</head>
+<body>
+
+    <ul id="nav-bar">
+        <li><a  href="#Agency banking">Agency Banking</a></li>
+        <li><a  href="#business content provider">Business content provider</a></li>
+        <li><a href="#sim unboarding">Sim unboarding</a></li>
+    </ul>
+
+    
+
+
+        <h1 id="header">
+            <p class="center">Fetch Business Hub</p>
+            
+                <img id="header-image" width="150px" src="image/WhatsApp Image 2021-07-15 at 09.55.05.jpeg" alt="Company logo">
+                      
+            
+            </h1>
```

Comment:
Please I think you should remove the white spaces at the start of your code lines

Reply:
- [ ]  For better communication Instead of giving orders, make suggestions. So that the student will be fully motivated to implement the changes you requested. You may even use this sentence `I think it would be better to remove the white spaces at the start of your code lines `